react/players/HTML5Player.tsx

```diff
@@ -1,16 +1,18 @@
 /* eslint-disable jsx-a11y/media-has-caption */
-import React, { useRef, FC } from 'react'
+import type { FC } from 'react'
+import React, { useRef } from 'react'
 import { useCssHandles } from 'vtex.css-handles'
 
 import useVideo from '../useVideo'
 import styles from '../styles/styles.css'
-import { VideoPlayer } from '../interfaces'
+import type { VideoPlayer } from '../interfaces'
 import PlayButton from '../controls/PlayButton'
 import FullscreenButton from '../controls/FullscreenButton'
 import TrackControl from '../controls/TrackControl'
 import VolumeControl from '../controls/VolumeControl'
+import { useVideoHandles, VideoHandlesProvider } from '../HandlesContext'
 
-const CSS_HANDLES = [
+export const CSS_HANDLES = [
```

Comment:
I think it would be more clear if each handle was exported on the component it's being used and then imported into this array. You could then see the whole picture when looking at a single component.